suuchi-rocksdb/src/test/scala/in/ashwanthkumar/suuchi/store/rocksdb/RocksDbStoreSpec.scala

```diff
@@ -2,29 +2,56 @@ package in.ashwanthkumar.suuchi.store.rocksdb
 
 import java.io.File
 
+import in.ashwanthkumar.suuchi.store.KV
 import org.apache.commons.io.FileUtils
-import org.scalatest.Matchers._
+import org.scalatest.Matchers.{be, convertToAnyShouldWrapper, have, startWith}
 import org.scalatest.{BeforeAndAfter, FlatSpec}
 
 class RocksDbStoreSpec extends FlatSpec with BeforeAndAfter {
-  val ROCKSDB_TEST_LOCATION = "/tmp/suuchi-rocks-test"
+
+  val ROCKSDB_TEST_LOCATION = "/tmp/suuchi-rocks-test/"
```

Comment:
@gsriram7 Instead of using this hard-coded path see if we can use the [`Files.createTempDirectory`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory(java.nio.file.Path,%20java.lang.String,%20java.nio.file.attribute.FileAttribute...). This would ensure we automatically choose a temp directory based on the platform.